modules/currency_denomination/includes/commerce_pos_currency_denomination.admin.inc

```diff
@@ -0,0 +1,359 @@
+      t('Delete'),
+      t('Cancel')
+    );
+  }
+  else {
+    form_set_error('', t('Invalid currency ID.'));
+  }
+
+  return $form;
+}
+
+/**
+ * Submit handler for deleting a denomination type.
+ */
+function commerce_pos_currency_delete_denomination_type_form_submit($form, &$form_state) {
+  if (isset($form_state['denomination_type_info'])) {
+    // First delete the denominations for that type.
+    db_delete('commerce_pos_currency_denomination')
+      ->condition('currency_id', $form_state['denomination_type_info']->currency_id)
+      ->execute();
+
+    // Now delete the denomination type.
+    db_delete('commerce_pos_currency_denomination_type')
+      ->condition('currency_id', $form_state['denomination_type_info']->currency_id)
+      ->execute();
+  }
+
+  drupal_set_message(t('The denomination type @type has been successfully deleted.', array(
+    '@type' => $form_state['denomination_type_info']->currency_code
+  )));
+  drupal_goto('admin/commerce/config/pos/denomination');
+}
+
+function commerce_pos_currency_view_denominations($currency_id) {
+  $output = '';
+
+  // Get the denomination type info.
+  $denomination_type_info = commerce_pos_currency_denomination_get_currency_types_from_id(array($currency_id));
+
+  if (!empty($denomination_type_info)) {
+    $denomination_type_info = $denomination_type_info[$currency_id];
+
+    $output .= l(t('Add Denomination'), 'admin/commerce/config/pos/denomination-type/' . $denomination_type_info->currency_id . '/denomination/add', array(
+      'attributes' => array('class' => array('add-denomination', 'button'))
+    ));
+
+    // Now, get the denominations for this denomination type.
+    $denominations = commerce_pos_currency_denomination_get_denominations(array($denomination_type_info->currency_code));
+    if (!empty($denominations)) {
+      $output .= t('<h3>Current Denominations for Type @type</h3>', array(
+        '@type' => $denomination_type_info->currency_code
+      ));
+
+      $headers = array(
+        'Title' => t('Title'),
+        'Amount' => t('Amount'),
+        'Options' => t('Options'),
+      );
+      $rows = array();
+
+      foreach ($denominations[$denomination_type_info->currency_id] as $denomination) {
+        $row = array();
+        $row[] = $denomination->denomination_title;
+        $row[] = $denomination->denomination_amount;
+        $row[] = l(t('delete'), 'admin/commerce/config/pos/denomination-type/' . $currency_id . '/denomination/' . $denomination->denomination_id . '/delete');
+        $rows[] = $row;
+      }
+
+      $output .= theme('table', array(
+          'header' => $headers,
+          'rows' => $rows,
+          'sticky' => TRUE,
+          'attributes' => array(
+            'class' => array('commerce-pos-current-denominations')
+          )
+        )
+      );
+    }
+    else {
+      $output .= t('<p>Currently, no denominations have been added for this denomination type.</p>');

```

Comment:
You could use the 'empty' parameter for theme_table for this if you want to simplify some of the logic here: https://api.drupal.org/api/drupal/includes%21theme.inc/function/theme_table/7.x